test/e2e/kube_metrics_adapter_test.go

```diff
@@ -397,14 +402,18 @@ func podHPA(deploymentName, name, apiVersion, kind string, metricTargets map[str
 		metrics = append(metrics, autoscaling.MetricSpec{
 			Type: autoscaling.ObjectMetricSourceType,
 			Object: &autoscaling.ObjectMetricSource{
-				MetricName: metric,
-				Target: autoscaling.CrossVersionObjectReference{
+				DescribedObject: autoscaling.CrossVersionObjectReference{
 					APIVersion: apiVersion,
 					Kind:       kind,
 					Name:       name,
 				},
-				TargetValue:  *resource.NewQuantity(target, resource.DecimalSI),
```

Comment:
This field is not necessary. Also, now its necessary to specify if its a "Value" or "AverageValue" as a metric target type.